crates/bevy_audio/src/audio_output.rs

```diff
@@ -79,7 +80,7 @@ where
             if let Some(audio_source) = audio_sources.get(&config.source_handle) {
                 if let Some(sink) = self.play_source(audio_source, config.settings.repeat) {
                     sink.set_speed(config.settings.speed);
-                    sink.set_volume(config.settings.volume);
+                    sink.set_volume(config.settings.volume * global_volume.volume);
```

Comment:
Should the `absolute_volume` flag be used here?

Reply:
Yeah, don't recommend submitting code late at night